relay-filter/src/lib.rs

```diff
@@ -20,6 +20,7 @@ pub mod browser_extensions;
 pub mod client_ips;
 pub mod csp;
 pub mod error_messages;
+pub mod generic_filters;
```

Comment:
nit: I would call this module, `generic` as we are already in the `relay_filter` crate. See https://doc.rust-lang.org/1.0.0/style/style/naming/README.html#avoid-redundant-prefixes-%5Brfc-356%5D.